frontend/Auth/FinishResetPassword.tsx

```diff
@@ -56,60 +60,102 @@ export function FinishResetPassword(props: RouterProps) {
 
   return (
     <div className="flex justify-center h-screen p-24 sm:pt-80">
-      <form
-        onSubmit={unary(submitMutation.mutate)}
-        className="space-y-32 relative lg:max-w-sm w-64"
-      >
-        <div>
-          <h1 className="text-gray-500 place-self-start mb-2 text-5xl lg:text-xl">
-            Set New Password
-          </h1>
-          <p className="text-left text-gray-600 py-8 text-4xl lg:text-sm">
-            Enter your new password below.
-          </p>
-          <FormField className="mb-4">
-            <FormFieldLabel className="text-3xl lg:text-xs" htmlFor="username">
-              Email
-            </FormFieldLabel>
-            <Input
-              id="username"
-              type="email"
-              placeholder={userEmail}
-              value={userEmail}
-              disabled
-              className="text-gray-500"
-            />
-          </FormField>
-          <FormField className="mb-4">
-            <FormFieldLabel className="text-3xl lg:text-xs" htmlFor="password">
-              Password
-            </FormFieldLabel>
-            <Input
-              id="password"
-              className="text-5xl lg:text-sm"
-              type="password"
-              value={finishResetPasswordFormData.password}
-              onChange={(evt) =>
-                setFinishResetPasswordFormData({
-                  ...finishResetPasswordFormData,
-                  password: evt.target.value,
-                })
-              }
-              required
-            />
-          </FormField>
-        </div>
+      {tokenValidationResponse?.tokenIsValid == true &&
```

Comment:
Always use triple equals in javascript, as a general rule. The reason is that double equals results in type coercion which is complicated enough for very few to fully understand. Triple equals avoids that complication

```suggestion
      {tokenValidationResponse?.tokenIsValid === true &&
```

Reply:
Thank you! Will do.